doc/source/style.rst

```diff
@@ -0,0 +1,25 @@
+.. _style:
+
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import numpy as np
+   import pandas as pd
+   np.set_printoptions(precision=4, suppress=True)
+   pd.options.display.max_rows = 15
+
+*******
+Styling
+*******
+
+.. versionadded:: 0.17.1
+
+
+You can apply **conditional formatting**, the visual styling of a DataFrame
+depending on the data within, by using the ``.style`` property.
+This is a property on every object that returns a ``Styler`` object, which has
+useful methods for formatting and displaying DataFrames.
+
```

Comment:
Was just looking into how git handled symlinks :) Probably best to just keep it here, perhaps with a link to download it. It actually integrates well with sphinx.

<img width="844" alt="screen shot 2015-11-15 at 12 12 17 pm" src="https://cloud.githubusercontent.com/assets/1312546/11170186/343d6254-8b92-11e5-8161-1133a57e5785.png">

I'll tweak a few things in our website CSS.